Documentation/contributing/testing/ci.rst

```diff
@@ -221,6 +224,16 @@ For a full list of Jenkins Jobs, see `Jenkins
 <https://jenkins.cilium.io/view/PR/>`_. Trigger phrases are configured within
 each job's build triggers advanced options.
 
+``test-older-k8s`` allows testing of all K8s versions 1.14 through 1.19 (it will trigger the appropriate ``Cilium-PR-K8s-X.XX-kernel-4.9`` jobs).
+If a specific K8s version fails, it can be re-ran using ``test-x.xx-4.9`` (with ``x.xx`` being the specific K8s version).
+
+``test-missed-k8s`` is deprecated and superseded by ``test-older-k8s``:
+
+- For new branches: it triggers a ``Cilium-PR-Ginkgo-Tests-K8s`` which does exactly the same thing (triggers all ``Cilium-PR-K8s-X.XX-kernel-4.9`` jobs),
+  except that the compound job will registered in PR checks. This makes it more cumbersome to detect which specific version K8s failed in case of failure.
```

Comment:
`the compound job will registered in PR checks` -> I'm not familiar enough with Jenkins to understand what this means. (+ typo `will register`/`will be registered`)

Reply:
I tried to reference the PR checks displayed directly on the PR page, e.g. if you look below, there is the "Some checks haven’t completed yet" block where one can check for status on jobs linked to this PR, which are triggered by `test-me-please` and alike.

How about this:
```
except that the compound job (and not the individual jobs) will be displayed on the PR page.
```

Reply:
Sounds better to me :+1: